tests/python/unittest/test_operator.py

```diff
@@ -30,6 +30,41 @@
 from common import setup_module, with_seed, teardown, assert_raises_cudnn_disabled, assertRaises
 import unittest
 
+@with_seed()
```

Comment:
I will move this back to its proper place - didn't get a chance as my family arrived home =P